core/activity/src/error.rs

```diff
@@ -115,7 +115,11 @@ impl ResponseError for Error {
             Error::Timeout => {
                 HttpResponse::RequestTimeout().json(ErrorMessage::new(self.to_string()))
             }
-            _ => HttpResponse::InternalServerError().json(ErrorMessage::new(self.to_string())),
+            _ => {
+                let e = self.to_string();
+                log::error!("Activity API server error: {}", e);
```

Comment:
this is needed, because I'm supressing `actix_http::response` logs
https://github.com/golemfactory/yagna/pull/715/files#diff-ff490246b403bbf6555dcfcf6cb38d31758d138d6fadbe6cd529de60d4374a99R384